data/json/items/archery.json

```diff
@@ -540,7 +542,40 @@
       "dispersion" : 120,
       "durability" : 6,
       "clip_size" : 1,
-      "reload": 40
+      "reload": 40,
+      "valid_mod_locations": [["dampening", 1]]
+   },
+   {
+      "id" : "hybridbow",
+      "type" : "GUN",
+      "symbol" : "(",
+      "color" : "yellow",
+      "name" : "hybrid longbow",
+      "description" : "A modernised, six-foot longbow that curves away from the archer at the tips for increased power. It can be used effectively by those of somewhat above-average strength. Designed for archers that appreciate the smoothness of a traditional longbow, but desire the superior speed of a modern recurve.",
```

Comment:
Two spaces after periods between the sentences.

Reply:
Damn. Thought i got all those.